### PR TITLE
Fix debug logs

### PR DESCRIPTION
**Pull request by bogdan-sava**

### CODE REVIEW

**Comment by bogdan-sava** [comment]
recheck

**Comment by bogdan-sava** [comment]
It is not added. It was there.
Basically there were two implementations for slf4j. Using log4j and logback.
Just picked logback as it is better in my opinion.

**Comment by cmgrote** [comment]
> It is not added. It was there.

Sorry, wasn't obvious from the diff but I see that now.

**Comment by mandy-chessell** [comment]
The logback library was already in use in the IV OMAS.  I moved it to the top level pom yesterday to clear the SLF errors in the build.  This seems to have impacted the Spring server chassis.

The EPL license is OK - I do not understand the implications of a dual license. - Will check with Steve.  It is OK to merge the change.

If debug coming out on stdout by default then need to fix that.  However, it is OK for debug to come out on stdout if explicitly enabled.

**Comment by bogdan-sava** [comment]
I don't think they were intentionally removed. They are debug log.  It was caused by the moving 2 dependencies ( log4j-to-slf4j and logback-classic ) in the parent pom to be available on all the modules.
Those 2 scope was test so they were only available for tests not included in build and no more logging lib for application.

File: pom.xml [hunk 1]
```diff
@@ -137,6 +137,11 @@
                 <artifactId>plexus-utils</artifactId>
                 <version>3.1.1</version>
             </dependency>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>1.2.3</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
```
Comment (mandy-chessell) [inline pom.xml]: Many dependencies appear to be outsince of the dependency management tag - is this right?

**Comment by bogdan-sava** [comment]
Both 
log4j-to-slf4j (logging bridge) and logback-classic are compile dependencies of spring-boot. 
Both were there since spring-boot is used. 
Only the dependency scope was changed and caused log missing.

It was a pure maven configuration issue.

File: pom.xml (hunk 1 above)
Comment (bogdan-sava) [reply pom.xml]: Yes. 
We start using it recently. DependencyManagement should be used carefully. It helps with the control of versions of artifacts used in transitive dependencies.

**Pull request merged**